### DISCUSSION TITLE
What's the meaning of "Blocked" in time chart of URL ?

### DISCUSSION THREAD

**Discussion by frankbernard** [MisterPhilip/omnibug#153] [Q&A] [answered]
If you open the URL then there is an item "Time" (in german "Zeit")
The first time bar is named "Blockiert", in english "blocked". That's before DNS lookup.
What exactly does that mean ?
Why is there a blocking before DNS lookup ?
Who blocks ? Who is responsible for that ?

Regards

Frank

**Comment by frankbernard** [comment]
![grafik](https://user-images.githubusercontent.com/89641157/131122121-37c8569a-5dcd-46bf-a062-2e40a294175b.png)
Example attached

**Comment by MisterPhilip** [comment, marked as answer]
This is typically due to other resources needing to load on the page first. I'm not exactly sure why you're seeing it for the initial page load, but Omnibug currently does not block any requests.
